build.wake

```diff
@@ -107,12 +107,17 @@ def doInstall dest kind =
     # kick off in parallel, but rsc and rsc_tool cannot be built at the same time right now.
     # Force them to be serial relative to each other.
     def install_rsc_bins =
+        require Pass rsc_migration = buildRSCMigration Unit
```

Comment:
I think the comment above should be updated to mention that RSCMigration is also part of the serial chain, or if thats not a constraint move the require on RSCMigraiton later.